engineering/README.md

```diff
@@ -0,0 +1,155 @@
+**Responsible party:** Pull request reviewer
+
+The reviewer of a pull request is moves the pull request and any issues that it satisfies to the Accepted column when the pull request is approved.
+
+Issues that don't pass review should be moved by the pull request reviewer back to the To Do column.
+
+
+## Pull Requests
+### Creating a pull request
+**Responsible party:** Issue assignee
+
+Pull requests are made from topic (feature, hotfix, etc) branches to `develop` or `master`. We never commit directly to these two branches.
+
+#### Checklist
+[ ] A description of what is included in the pull request
+[ ] Links to any other pull requests in other repos that are required for this pull request
+[ ] Instructions on how to test this pull request
+[ ] Links to the github issue(s) this pull request addresses
+[ ] Assigned to & review request from the person responsible for acceptance testing
+[ ] Tests written for all new or modified functionality
+[ ] Lint free
+[ ] Type declarations written for all modified files (JavaScript)
+[ ] Comments for any public functions
+
+*After successful review, the assignee merges the PR.*
+
+## Reviewing a pull request
+
+**Responsible party:** Reviewer 
+
+Pull request reviews are conducted when a pull request is made from a feature or hotfix branch to `develop` or `master`. We do a single code review by a colleague who also works on the same codebase for most things. For anything that involves cryptography, secrets or consensus, we require at least 2 reviews by different people.
+
+#### Review checklist
+
+Below is a basic checklist of items that should be checked during a pull request code review.
+
+- [ ] Does the code work and does it perform its intended function?
+- [ ] Is all the code easily understood? Do variable and function names clearly communicate their function?
+- [ ] Is there properly formatted (eg. jsdoc for JavaScript) documentation for public and private API methods?
+- [ ] Does any code duplicate functionality?
+- [ ] Are there any global variables that can be removed?
+- [ ] Is there any commented out code?
+- [ ] Is there any logging or debugging code that isn't using a logger that can be removed?   
+- [ ] Do tests exist and do they adequately cover the changed behavior?
+- [ ] Is the code being tested by any linter or type checking tools and does it pass?
+
+
+## Repository management
+We generally use the [git flow](http://nvie.com/posts/a-successful-git-branching-model/) git repository workflow. 
+
+The `develop` branch contains the latest delivered and accepted features. New features are generally developed against the `develop` branch unless they depend on the work done in another feature branch that has yet to be accepted and merged into `develop.
+
+## Planning
+[ ] Before Friday Engineering meeting: Each team member collects notes on their top priorities for the coming week, progress on existing items, concerns etc. and adds bullet list items to Friday engineering meeting agenda.
```

Comment:
Friday or Thursday?  Do we want to say 9am EST/EDT in the doc?  Do we want to link to the forum in this doc?

Reply:
Also, should we have a template for this?  Example:

```
Topic: <name of your topic>
Owner: <your name>
Priority: <low|medium|high>
Description: <a few sentences describing the concern>
Desired outcome: <what needs to be decided from this discussion>
```

Reply:
Yes! great ideas!